src/install.html

```diff
@@ -88,20 +88,19 @@
             />
           </div>
           <div class="col-md-8 order-md-1 col-lg-7 text-center text-md-start">
-            <h1 class="mb-3">Install Artichoke Ruby</h1>
+            <h1 class="mb-3"><%= it.t.install.title %></h1>
             <p class="lead mb-4">
-              Artichoke Ruby installs on Linux, macOS, and Windows. New builds
-              are pushed nightly.
+              <%= it.t.install.install %>
             </p>
             <p class="lead mb-4">
-              Try out a build using one of these installation methods.
+              <%= it.t.install.try %>
             </p>
             <div class="d-flex flex-column flex-md-row">
               <a
                 href="#nightly"
                 class="btn btn-lg btn-artichoke-primary mb-3 me-md-3"
               >
-                Nightly Builds
+                <%= it.t.install.nightly.go %>
```

Comment:
do the `Docker` and `Cargo` go buttons not get localized because they are names of tools? If these buttons were to change, they'd have to be internationalized in the HTML source, right?